ChurchCRM/CRM
# Add support for default Zip Code

# Description & Issue number it closes 


Adds support for a default Zip Code.

Resolves #6951

## Screenshots (if appropriate)
 
![Screen Shot 2024-04-23 at 21 28 42](https://github.com/ChurchCRM/CRM/assets/201834/2a5bb5da-fa9c-4418-9ad0-27cc416ea396)

![Screen Shot 2024-04-23 at 21 29 26](https://github.com/ChurchCRM/CRM/assets/201834/a9ef890d-9c7b-4c00-8836-720d63f172d6)


## How to test the changes?

- Check that the field is in `/SystemSettings.php#PeopleSetup`
- Typed in a value & save
- Go to `/FamilyEditor.php` & confirm that the field is pre-populated.

## Type of change


# How Has This Been Tested?



- Visual inspection.
- Check that the field was in `/SystemSettings.php`
- Typed in a value & saved
- Checked that the value was saved in the DB.
- Went to `/FamilyEditor.php` & confirmed that the field was pre-populated.

Changes applied and tested on live site.

# Checklist:
